DOCS/interface-changes.rst

```diff
@@ -70,6 +70,8 @@ Interface changes
       pad must be connected either to another filter, or to a video/audio track
       or video/audio output). If they are disconnected at runtime, the stream
       will probably stall.
+    - deprecate the OpenGL cocoa backend, option choice --gpu-context=cocoa
```

Comment:
Sorry, I thought I left a comment here yesterday, but it seems to have disappeared. Is it worth noting something about this in the manpage?